### DISCUSSION TITLE
Variable Pointer

### DISCUSSION THREAD

**Discussion by vgauti01** [Guizz27/DWIN_LK4Pro#16] [Q&A]
Hello,
first I want to thank you for all the work you have done, my LK4 Pro works perfectly.
I am now trying to make my version. But there is something I don't understand with DGUS Tools. What
what the VP (Variable pointer) and how to know which 0xXXXX corresponds to which real Marlin variable? How to know that one variable is the temperature of the nozzle, the other the power of the fan...
Thank you in advance for your answer.

**Comment by Guizz27** [comment]
You can try to read the documentation http://www.ampdisplay.com/documents/pdf/DWIN%20DGUS%20DEV%20GUIDE_V40_2014.pdf

VP is an address in memory of the screen. 

All the mapping between VP and Marlin happens in Marlin in Marlin\src\lcd\extui\dgus_reloaded.
